ngraph/src/ngraph/runtime/reference/prior_box_clustered.hpp

```diff
@@ -0,0 +1,116 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <cmath>
+
+#include "ngraph/axis_vector.hpp"
+#include "ngraph/check.hpp"
+#include "ngraph/coordinate_transform.hpp"
+#include "ngraph/op/prior_box_clustered.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            template <typename T>
+            void prior_box_clustered(const T* data,
+                                     const T* img,
+                                     float* dst_data,
+                                     const Shape& out_shape,
+                                     const op::PriorBoxClusteredAttrs& attrs)
+            {
+                int num_priors_ = attrs.widths.size();
+
+                auto variances = attrs.variances;
+                if (variances.empty())
+                    variances.push_back(0.1f);
+
+                // Execute
+                const int layer_width = data[1];
+                const int layer_height = data[0];
+
+                int img_width = img[1];
+                int img_height = img[0];
+
+                // TODO: Uncomment after PriorBoxClustered is aligned with the specification.
+
+                //                int img_width = img_w_ == 0 ? img[1] : img_w_;
+                //                int img_height = img_h_ == 0 ? img[0] : img_h_;
+
+                //                float step_w = attrs.step_widths == 0 ? step_ : attrs.step_widths;
+                //                float step_h = attrs.step_heights == 0 ? step_ :
+                //                attrs.step_heights;
+
+                float step_w = attrs.step_widths == 0 ? 0 : attrs.step_widths;
+                float step_h = attrs.step_heights == 0 ? 0 : attrs.step_heights;
```

Comment:
Looks like it can be changed to:
```suggestion
                float step_w = attrs.step_widths;
                float step_h = attrs.step_heights;
```